rn/Teacher/src/routing/register-screens.js

```diff
@@ -156,7 +155,7 @@ export function registerScreens (store: Store): void {
   registerScreen('/filter', wrap(Filter), store)
   registerScreen('/to-do', wrap(ToDoList), store, { canBecomeMaster: true, deepLink: true })
   registerScreen('/courses/:courseID/wiki', wrap(PageDetails), store, { deepLink: true })
-  registerScreen('/courses/:courseID/pages', wrap(PagesList), store, { canBecomeMaster: true, deepLink: true })
+  registerScreen('/courses/:courseID/pages', null, store, { canBecomeMaster: true, deepLink: true })
```

Comment:
This isn't quite ready for Teacher app yet so you need to move this line to the `if (isStudent()) {` check around like 192 and add the old line back but put it in the `if (isTeacher()) {` check around line 173.